cogip/entities/robot_manual.py

```diff
@@ -28,7 +28,7 @@ def __init__(
             self,
```

Comment:
Huge commit ! I trust you, just checking typo ! ;)
Just a question: do you think saving camera placement could be possible when you close/open again the monitor ?

Reply:
About the camera position save/restore: I added it, which required to also add a shortcut to restore the default view (Return key).

Reply:
Ah ! top, I missed that !